### ISSUE TITLE
groovy.lang.MissingPropertyException: No such property: Status for class: java.util.LinkedHashMap$Entry

### ISSUE THREAD

**Issue by kdebski85** [avast/gradle-docker-compose-plugin#420]
The plugin is working fine for me, but there is an error displayed:

```Starting process 'command 'docker''. Working directory: /Users/xyz/IdeaProjects/abc Command: docker compose -f compose.yml --ansi never -p fa2a4756e42e314ce481d9425735a88d_abc_ ps --format json
Successfully started process 'command 'docker''
Cannot process JSON returned from 'docker compose ps --format json'
groovy.lang.MissingPropertyException: No such property: Status for class: java.util.LinkedHashMap$Entry
        at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.unwrap(ScriptBytecodeAdapter.java:65)
        at org.codehaus.groovy.runtime.callsite.GetEffectivePojoPropertySite.getProperty(GetEffectivePojoPropertySite.java:65)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:329)
        at com.avast.gradle.dockercompose.tasks.ComposeUp$_getStateForCache_closure12.doCall(ComposeUp.groovy:220)
```

---

My settings:

```
dockerCompose {
    useComposeFiles = ['compose.yml']
    stopContainers = false
    useDockerComposeV2 = true
}
```

---

plugin version: 0.17.4
Gradle version: 7.5.1
Java version: 17.0.8
Groovy version: 4.0.14
System: MacOS with M1 processor
Docker version: 24.0.6 with Colima 0.5.5

---

`compose.yml ` contains 3 services. 
When I run the command manually, the output has `Status` element for each service (I removed other properties):
The output is not really a JSON, but 3 JSONs concatenated:

```
{
  "Command": "\"docker-entrypoint.sh command1\"",
  "Status": "Up 17 minutes"
}
{
  "Command": "\"docker-entrypoint.sh command2\"",
  "Status": "Up 17 minutes"
}
{
  "Command": "\"docker-entrypoint.sh command3\"",
  "Status": "Up 17 minutes"
}
```

**Comment by Krzysztof-Debski** [comment]
From what I tested, the plugin assumes that `ps` output will be JSON array format:
It works fine, for something like this:
```
[
    {
        "Command": "docker-entrypoint.sh command1",
        "Status": "Up 17 minutes"
    },
    {
        "Command": "docker-entrypoint.sh command2",
        "Status": "Up 17 minutes"
    },
    {
        "Command": "docker-entrypoint.sh command3",
        "Status": "Up 17 minutes"
    }
]
```

Is it Docker issue that the format is wrong?

**Comment by kdebski85** [comment]
From what I tested, the plugin assumes that `ps` output will be JSON array format:
It works fine, for something like this:
```
[
    {
        "Command": "docker-entrypoint.sh command1",
        "Status": "Up 17 minutes"
    },
    {
        "Command": "docker-entrypoint.sh command2",
        "Status": "Up 17 minutes"
    },
    {
        "Command": "docker-entrypoint.sh command3",
        "Status": "Up 17 minutes"
    }
]
```

Is it Docker issue that the format is wrong?

**Comment by kdebski85** [comment]
Please see https://stackoverflow.com/questions/55583208/docker-and-format-json
docker `json` format returns `json` for each line, but the whole output is not `json` array.

The plugin should treat each line as a separate json.

**Comment by augi** [comment]
Addressed in the 0.17.5 release, thanks for your reports and information provided! 👍
